csharp/test/Apache.Arrow.Tests/SchemaBuilderTests.cs

```diff
@@ -16,12 +16,36 @@
 using Apache.Arrow.Types;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using Xunit;
 
 namespace Apache.Arrow.Tests
 {
     public class SchemaBuilderTests
     {
+        public class SchemaComparer        {
```

Comment:
braces go on a newline:

(1) in [https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)